content/en/docs/architecture/branching.md

```diff
@@ -6,6 +6,50 @@ description: How OCP component repositories have their content and configuration
 # Motivation
 The number of repositories and branches involved in assembling an OpenShift release (meaning here a colloquialism, more than the strict set of repositories needed to build the images in a release payload) is large. These repositories need to move at the same cadence and branch and configure at the same cadence when release cuts are made. DPTP owns the tooling that makes these cuts simple.
 
+# Branching Phases
+Every OCP release travels through four phases - normal development, feature freeze, code freeze, and general availability. This section details what to expect from each of these phases and the types of merge requirements that exist for PRs in each phase.
+
+## Normal Development For 4.x Release
+During normal development, active development branches are open to all changes.
+
+|*Branch*|*Promotes To*|*Merge Criteria*|*Fast-Forwarded?*|
+|-|-|-|-|
+|`master/main`|`ocp/4.x`|`lgtm`, `approved`|no|
+|`release-4.x-n`|`ocp/4.x-n`|`lgtm`, `approved`, `bugzilla/valid-bug`, `cherry-pick-approved`|no|
+|`release-4.x`|nowhere|merges blocked|yes|
+|`release-4.x+1`|`ocp/4.x+1`|merges blocked|yes|
+
+## Feature Freeze For 4.x Release
+During feature freeze, the expectation is that merges to the active development branch are fixing bugs, so we require that all PRs that land have a valid bug label.
+
+|*Branch*|*Promotes To*|*Merge Criteria*|*Fast-Forwarded?*|
+|-|-|-|-|
+|`master/main`|`ocp/4.x`|`lgtm`, `approved`, `bugzilla/valid-bug`, `cherry-pick-approved`|no|
```

Comment:
I'm now not exactly sure but it's definitely not `cherry-pick-approved`, it's either `group-lead-approved` or just valid bug.

Reply:
I'm copying the doc, which got approved by @eparis  and @jupierce

Reply:
After feature freeze we should have valid bug, but do not need cp-approve or group-lead-approve. So I think this is right.

Reply:
So the table is incorrect right now b/c it says `cherry-pick-approved` is needed and it is really not?